doc/Makefile

```diff
@@ -42,18 +46,27 @@ info: $(addprefix ../info/,$(SGMLS:.sgml=.info))
 ../info/%.info: %.sgml | ../info
 	@cd ../info && linuxdoc -B info ../doc/$<
 
+../tex/%.tex: %.sgml | ../tex
+	@cd ../tex && linuxdoc -B latex ../doc/$<
+
+../tex/%.dvi: ../tex/%.tex | ../tex
+	@cd ../tex && (latex $< && latex $< && latex $< && latex $<)
```

Comment:
I consider TEX and DVI files both as intermediate files ATM. I didn't want to make another directory to hold the DVI files. So both of them reside in the "tex" directory. This could be changed but I don't see the point in it besides being "100% correct". The flow is "*.sgml -> *.tex -> *.dvi -> *.pdf".

Reply:
If both the .tex and .dvi files are temporary files then there shouldn't be a ../tex directory in the first place, should it? Wouldn't it be "100% correct" to have a ../docwrk or ../docwrk/tex directory (analog to wrk and libwrk).

Reply:
Can you please try to explain what the `(latex $< && latex $< && latex $< && latex $<)` does?

Reply:
latex is weird like that, sometimes you have to run it more than once on the same input to produce proper output

Reply:
It's because multipass compilation is NOT automatic. It recognises the need for another pass and informs the user but does not do it by itself. FWIW I wouldn't call it "weird" ;-)

Reply:
I also believe there should be a more clean and robust way of getting correct output than throwing a – seemingly arbitrary – number of invocations in hope that it will be enough.